src/pages/_app.jsx

```diff
@@ -22,10 +22,16 @@ const MyApp = ({ Component, pageProps, router, currentUser }) => (
       <title>Hope Sustains Life</title>
     </Head>
     <div className="App">
-      {!router.pathname.includes("admin") && <NavBar />}
+      {!(
+        router.pathname.includes("admin") ||
+        router.pathname.includes("resetpassword")
+      ) && <NavBar />}
       <div className="Content">
         <Component {...pageProps} currentUser={currentUser} />
-        {!router.pathname.includes("admin") && (
+        {!(
+          router.pathname.includes("admin") ||
+          router.pathname.includes("resetpassword")
```

Comment:
Pull this logic (checking if the route is "admin" or "resetpassword") into a function that iterates through a list of route names, preferably using the values found in `urls.js`